### DISCUSSION TITLE
Assign Tasks to Specific user

### DISCUSSION THREAD

**Discussion by MohamedAgiez** [flowsforapex/apex-flowsforapex#645] [General] [answered]
Is it possible to assign User Task to specific user and Later filter inbox based on given user.
some thing like Oracle BPM behaviour

**Comment by rallen2010** [comment, marked as answer]
This is indeed one of the features in our upcoming Flows forAPEX v23.1 release, which will be available very, very  soon.

**Reply by MohamedAgiez** [reply]
Hope this will be very very soon :)

**Reply by rallen2010** [reply]
Released today.  Hope you will like it!
